src/cljs/rems/atoms.cljs

```diff
@@ -7,6 +7,9 @@
 (defn external-link []
   [:i {:class "fa fa-external-link-alt"}])
 
+(defn more-options []
+  [:i {:class "fa fa-ellipsis-h"}])
```

Comment:
These icons could also be replaced with a CSS class. What do you think?

Reply:
I think using CSS for these things makes it more magic than it needs to be.